src/app/application.cpp

```diff
@@ -463,13 +463,22 @@ void Application::sendNotificationEmail(const BitTorrent::Torrent *torrent)
                      content);
 }
 
+void Application::torrentAdded(BitTorrent::Torrent *const torrent)
+{
+    Preferences *const pref = Preferences::instance();
```

Comment:
```suggestion
    const Preferences *pref = Preferences::instance();
```

Reply:
Changed.
But then again, the torrentFinished event just below uses it as constant data, so that we again create divergence here.